lib/fastlane/actions/update_url_schemes.rb

```diff
@@ -0,0 +1,68 @@
+require 'plist'
+
+module Fastlane
+  module Actions
+    class UpdateUrlSchemesAction < Action
+      def self.run(params)
+        path = params[:path]
+        url_schemes = params[:url_schemes]
+
+        hash = Plist.parse_xml(path)
+        hash['CFBundleURLTypes'].first['CFBundleURLSchemes'] = url_schemes
```

Comment:
I might be wrong, but couldn't this be done using `defaults write ...` to not require manual plist parsing.